### ISSUE TITLE
Toast: a11y – Screen Reader announces 'Close This Message' twice in Safari/VoiceOver

### ISSUE THREAD

**Issue by artemtrusov-cengage** [cengage/react-magma#1606]
**Describe the bug**
When a toast notification is triggered, VoiceOver in Safari reads 'Close this message' twice

**To Reproduce**
1. Go to the [Toast API](https://react-magma.cengage.com/version/4.6.0/api/toast/#toasts_container_(handling_multiple_toasts))
2. Turn on VoiceOver
3. Click the 'Show Toast' button
4. Notice that the issue occurs

**Expected behavior**
It should read it only once.

**Screenshots**
<img width="1440" alt="Screenshot" src="https://github.com/user-attachments/assets/44fbf618-68fa-4db8-b4d1-a3e24527d90f" />

**Desktop (please complete the following information):**
 - OS: macOS 15.1.1
 - Browser Safari 18.1.1

**Comment by orion-cengage** [comment]
Bug found in Firefox on Mac using Voiceover:

The original bug of Safari + Voiceover reading the aria-label text twice, has now become Firefox + Voiceover reading the title attribute twice. This was not caught when originally testing the PR.

**Comment by orion-cengage** [comment]
We have decided that we are not going to continue to find a work around for this issue. Using the title attribute instead of aria-label was acceptable in theory only because we thought it fixed the issue. But it turns out that it doesn't. This exact issue exists in other design systems.

We will mark this issue has "unable to fix" and we need to revert the PR so we can go back to using aria-live.

**Comment by orion-cengage** [comment]
Upon further review we have decided to move forward with this update. It fixes the issue with Safari + Voiceover, but the problem still remains with Firefox + Voiceover. One less browser/screenreader combo with the issue is enough for us to call this a win, and there's no further solution in sight for Firefox. 

This has also passed all other tests of browsers + NVDA and even JAWS.
